### PR TITLE
[OnPrem][TE-9740] Git Action file.

### PR DESCRIPTION
**Pull request by ManoharKrishnaBoppe**

### CODE REVIEW

File: .github/workflows/automation.yml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: "0"
+
+      - name: Set environment variables
+        run: |
+          echo "DEV_ADDON_URL=https://07e8-122-172-87-149.ngrok-free.app" >> $GITHUB_ENV
```
Comment (vikram-chaitanya) [inline .github/workflows/automation.yml]: Correct the URL's properly..Also add https://development-kibbutz.testsigma.com and https://vikings-kibbutz.testsigma.com and app URL is https://kibbutz.testsigma.com

File: .github/workflows/automation.yml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            git config --global user.name "${{ github.actor }}"
+            git branch -a   
+            git checkout $GIT_BRANCH
+            git add .
+            git commit -m "build files added to the path $version_dir"
+            git push origin $GIT_BRANCH
+            cd $version_dir
+            echo "Updating the $dir addon in stacks"
+            curl --location --request PUT '${{ env.DEV_ADDON_URL }}/api/v1/plugins' \
+            --header 'Authorization: Bearer ${{ secrets.DEV_V1_TOKEN }}' \
```
Comment (vikram-chaitanya) [inline .github/workflows/automation.yml]: is DEV_V1_TOKEN same for all envs?

**Review by vikram-chaitanya — CHANGES_REQUESTED** [review]

File: .github/workflows/automation.yml (hunk 2 above)
Comment (ManoharKrishnaBoppe) [reply .github/workflows/automation.yml]: @vikram-chaitanya No, we need add different based on the user in each stack.

File: .github/workflows/automation.yml (hunk 1 above)
Comment (ManoharKrishnaBoppe) [reply .github/workflows/automation.yml]: Corrected the urls @vikram-chaitanya

**Review by vikram-chaitanya — APPROVED** [review]

**Pull request merged**